js/web/lib/onnxjs/attribute.ts

```diff
@@ -181,8 +181,7 @@ export class Attribute {
       // format strings attributes are returned as string[], so no conversion is needed.
       if (attr instanceof onnx.AttributeProto) {
         const utf8Strings = value as Uint8Array[];
-        return utf8Strings.map(
-            utf8String => Buffer.from(utf8String.buffer, utf8String.byteOffset, utf8String.byteLength).toString());
+        return utf8Strings.map(decodeUtf8String);
```

Comment:
Do we not have a model that uses a string attribute in web testing ? It seems like if we had and Buffer is not supported on browsers, this issue would have been caught. If we don't have such a model, can we add one ?

Reply:
I removed polyfill for 'Buffer' for test (with some changes in test code to support the change).